services/blockchain-connector/tests/constants/blocks.js

```diff
@@ -13,80 +13,174 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const block = {
+const blockWithTransaction = {
```

Comment:
This change breaks functional test for encoder: `services/blockchain-connector/tests/functional/encoder.test.js`. Please re-verify.